### DISCUSSION TITLE
Nextjs custom domain sometimes works and sometimes it doesn't

### DISCUSSION THREAD

**Discussion by Aycom366** [vercel/community#1647] [Help]
### Summary

so I have a domain name I already got from Namecheap, and after I hosted my app on Vercel, I configured the ARecords and CName.
So sometimes the website works and sometimes it doesn't work and this has been a nightmare for me. Have contacted NameCheap about this and I was told the issue is not from their side and Vercel should be contacted for this.
So I don't know what am doing wrong and why this is happening.
Kindly help. Thanks

below is Vercel's webserver
[vercels](https://ecommerce-merchant-v2.vercel.app/)

below is the domain name
[shopazhub](http://www.useshopazhub.com)

**Comment by amyegan** [comment]
Hi @Aycom366. Can you tell me more about what happens when the custom domain doesn't work for you? I don't see a problem with it right now

**Reply by Aycom366** [reply]
Yeah everything works now, but sometimes it doesn't work. Like the IP address sometimes fails to connect.
Imagine a scenario where by am able to access YouTube on my side but you can't access it on your end and you have full excellent internet connection.
Infact most of the times I start asking people around if they can access the website and they will tell me it's going through meanwhile It not going on my end.

So am not saying it doesn't work but sometimes it fails

**Comment by Aycom366** [comment]
![image](https://user-images.githubusercontent.com/42998943/222259286-b525f099-06f7-419f-98d7-b7c3e2534889.png)
check this site here [zeeh afruca](www.zeeh.africa). Notice that it failing on my side.
this is what we have been experiencing. Please what can we do? @amyegan
